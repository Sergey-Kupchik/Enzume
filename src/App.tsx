import React from 'react';
import logo from './logo.svg';
import './App.css';

function App() {
  return (
    <div data-test="component-app"> 
add two
    </div>
  );
}

export default App;
