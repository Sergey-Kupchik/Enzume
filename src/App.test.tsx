import React from 'react';
import Enzyme, {shallow} from 'enzyme';
import EnzymeAdapter from '@wojtekmaj/enzyme-adapter-react-17';
import App from './App';


Enzyme.configure({adapter: new EnzymeAdapter()})
test('renders without crashing', () => {
  const warpper = shallow(<App/>)
  console.log(warpper.debug());
});

test('renders non-empty component without crashing', () => {
  const warpper = shallow(<App/>)
 expect(warpper.exists()).toBe(true)
});


test('renders without error', () => {
  const warpper = shallow(<App/>)
  const appComponent = warpper.find('[data-test="component-app"]')
  expect(appComponent.length).toBe(1)
});

test('renders increment button', () => {
  const warpper = shallow(<App/>)

});

test('renders counter dispay', () => {
    const warpper = shallow(<App/>)
  
  });

  test('counter dispay starts at 0', () => {
    const warpper = shallow(<App/>)
  
  });

  test('clicking button increments counter dispay', () => {
    const warpper = shallow(<App/>)
  
  });