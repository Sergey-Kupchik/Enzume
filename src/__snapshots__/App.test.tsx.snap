// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`clicking button increments counter dispay 1`] = `ShallowWrapper {}`;
